server/server.js

```diff
@@ -85,33 +87,36 @@ const startIntervals = (socket, stockName, timeRange) => {
   topPeers(socket, stockName, HOST, TOKEN);
   chartData(socket, stockName, timeRange, HOST, TOKEN);
   sectorInformation(socket, stockName, HOST, TOKEN);
-}
+};
 
 const getCompanySymbols = async () => {
   try {
     const companySymbols = await axios.get(
       `${HOST}/ref-data/symbols?token=${TOKEN}`
-    )
-
-    return companySymbols.data.map(data => ({ symbol: data.symbol, name: data.name }))
+    );
 
+    return companySymbols.data.map(data => ({
+      symbol: data.symbol,
+      name: data.name
+    }));
   } catch (error) {
     console.error(`Error: ${error}`);
   }
 };
 
 const searchQuery = async (socket, inputQuery, allSymbols) => {
   try {
-    const a = await allSymbols
-    const b = a.map(data => ({ symbol: data.symbol, name: data.name }))
-    const filteredData = b.filter(search => search.symbol.toLowerCase().indexOf(inputQuery.toLowerCase()) !== -1 || search.name.toLowerCase().indexOf(inputQuery.toLowerCase()) !== -1);
-
-
-    const topTen = filteredData.slice(0, 10)
+    const a = await allSymbols;
+    const b = a.map(data => ({ symbol: data.symbol, name: data.name }));
```

Comment:
nit: Better variable names than `a` and `b`, please!

Reply:
will change it